alvr/client_core/src/platform/mod.rs

```diff
@@ -69,6 +71,7 @@ pub fn platform() -> Platform {
             ("Oculus", _, "seacliff") => Platform::QuestPro,
             ("Oculus", _, _) => Platform::QuestUnknown,
             ("Pico", "Pico Neo 3", _) => Platform::PicoNeo3,
+            ("Pico", "Pico Neo3 Link", _) => Platform::PicoNeo3Link,
             ("Pico", _, _) => Platform::Pico4,
```

Comment:
Question to Neo 3 owners, is the model really displayed as "Pico Neo 3" in the settings? The "Pico Neo3 Link" is missing the space before the number. I also think a fallback should be an already existing model and not a newly added one.

Reply:
PLYSHKA responded to this question in Discord. The Neo 3 really has a space in front of the number in the model name.